frontend/src/app/render_wasm/serializers.cljs

```diff
@@ -0,0 +1,251 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; Copyright (c) KALEIDOS INC
+
+(ns app.render-wasm.serializers
+  (:require
+   [app.common.uuid :as uuid]
+   [cuerdas.core :as str]))
+
+(defn u8
+  [value]
+  (let [u8-arr (js/Uint8Array. 1)]
+    (aset u8-arr 0 value)
+    u8-arr))
+
+(defn f32->u8
+  [value]
+  (let [f32-arr (js/Float32Array. 1)]
+    (aset f32-arr 0 value)
+    (js/Uint8Array. (.-buffer f32-arr))))
+
+(defn i32->u8
+  [value]
+  (let [i32-arr (js/Int32Array. 1)]
+    (aset i32-arr 0 value)
+    (js/Uint8Array. (.-buffer i32-arr))))
+
+(defn bool->u8
+  [value]
+  (let [result (js/Uint8Array. 1)]
+    (aset result 0 (if value 1 0))
+    result))
+
+(defn uuid->u8
+  [id]
+  (let [buffer (uuid/get-u32 id)
+        u32-arr (js/Uint32Array. 4)]
+    (doseq [i (range 0 4)]
```

Comment:
Do it statically instead of range + doseq

Reply:
ok